tests/test_generate_hamilton_input_quant_studio.py

```diff
@@ -0,0 +1,138 @@
+from itertools import cycle
+
+from scripts.generate_hamilton_input_quant_studio import GenerateHamiltonInputQuantStudio
+from tests.test_common import TestEPP, FakeEntitiesMaker
+
+
+class TestGenerateHamiltonInputQuantStudio(TestEPP):
+    def setUp(self):
+        # argument -d left blank to write file to local directory
```

Comment:
There is no -d argument in `GenerateHamiltonInputQuantStudio`

Reply:
updated